### PR TITLE
[2.6] Add resource search back

### PR DESCRIPTION
**Pull request by nwmac**
Added as search icon in header bar
Keyboard shortcut still supported
Also fixes a bug with TLS certificates (in secrets.js)

### CODE REVIEW

File: components/nav/Header.vue [hunk 1]
```diff
@@ -322,34 +368,8 @@ export default {
...
-
-    > .back,
-    > .import,
-    > .kubectl {
-      text-align: right;
-
-      .btn {
-        text-align: center;
-      }
+    > .resource-search {
```
Comment (nwmac) [inline components/nav/Header.vue:371]: This can go

File: components/nav/Header.vue [hunk 2]
```diff
@@ -51,7 +55,12 @@ export default {
       const name = this.currentProduct.name;
 
       return this.$store.getters['i18n/withFallback'](`product."${ name }"`, null, ucFirst(name));
-    }
+    },
+
+    showSearch() {
+      return this.currentProduct?.name === EXPLORER;
```
Comment (nwmac) [inline components/nav/Header.vue:61]: Does not show with Apps and Marketplace and other products, where it should.

File: components/nav/Header.vue [hunk 3]
```diff
@@ -125,12 +136,31 @@ export default {
...
+        ref="searchButton"
+        v-shortkey="{windows: ['ctrl', 'k'], mac: ['meta', 'k']}"
+        type="button"
+        class="btn role-tertiary resource-search"
+        @shortkey="openSearch()"
+        @click="openSearch()"
+      >
+        <i v-tooltip="t('nav.resourceSearch.label')" class="icon icon-search icon-lg" />
+      </button>
+      <modal
```
Comment (nwmac) [inline components/nav/Header.vue:155]: Group headers are selectable, when they should not be

File: components/nav/Header.vue (hunk 1 above)
Comment (nwmac) [reply components/nav/Header.vue:371]: Done

**Review by richard-cox — APPROVED** [review]

**Pull request merged**